mailbox/backup/pom.xml

```diff
@@ -80,6 +123,11 @@
             <artifactId>junit-platform-launcher</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.projectreactor</groupId>
+            <artifactId>reactor-test</artifactId>
```

Comment:
Is this used?

Reply:
Good point, it was used in an iteration of JAMES-2681. But removed later. I will check in JAMES-2681 if i have removed it from the pom.xml.

Reply:
Just fixed in JAMES-2681 :)